src/classes/PS_IntegrationServiceConfig_TEST.cls

```diff
@@ -146,6 +146,25 @@ public with sharing class PS_IntegrationServiceConfig_TEST {
         System.assertEquals(testMerchantId, service.getMerchantIds());
     }
 
+    @IsTest
+    private static void shouldReturnViewCommitmentURL() {
+        String commitmentId = '11fe3d30-2191-4db7-bcd7-d069fa9f2add';
+        String suffix = '/admin/recurring/' + commitmentId;
+
+        PS_IntegrationService.setConfiguration(testConfig);
+
+        Test.startTest();
+        PS_IntegrationServiceConfig.Service service = new PS_IntegrationServiceConfig.Service();
+        Test.stopTest();
+
+        String commitmentViewURL = service.getCommitmentViewURL(commitmentId, PS_IntegrationServiceConfig.VIEW_TYPE.COMMITMENT);
+        String testCommitmentViewURL = testBaseUrl.replace('https://', 'https://admin.') + suffix;
+
+        System.debug(commitmentViewURL);
```

Comment:
- Let's remove debug statement.
- Let's rename `testCommitmentViewURL` with `expectedURL`.
- Let's remove the `suffix` var and construct the `String expectedURL = testBaseUrl.replace('https://', 'https://admin.') + "/" PS_IntegrationServiceConfig.VIEW_TYPE_COMMITMENT_SUFFIX +/" + commitmentId; ` 
-`System.assertEquals(expectedURL, commitmentViewURL, 'Commitment View URL should match');`.